packages/bpk-component-calendar/src/BpkCalendarGrid.module.scss

```diff
@@ -20,7 +20,6 @@
 @use '../../unstable__bpk-mixins/utils';
 
 .bpk-calendar-grid {
-  width: 100%;
```

Comment:
Be careful with removing this and making sure to test this as this can cause alignment issues across browser and also with the `fixed-size` styles too, especially with the generation of the scrollable calendar and datepicker

Reply:
Is there an exhaustive list that I can look at for manual testing? It seemed that `width: 100` previously was _always_ being overriden. This is applied to a `div` which also has `width: 100%` as default anyway, so I think it's safe, but I'll carry out any testing that you can think of to make sure this is safe. I understand it's a really complex component.

Reply:
Sadly no list but I know for the calendar yes this was always being overriden as I believe it sets the width to a fixed one with a class, but if you change it here the knockon affect for this to a consumer can be a broken calendar, so maybe with this component we need to do a test in a consumer before we merge this just for sanity sake as this is the monster component

Reply:
Note: tested on product and posted results offline 👍